app/contributing/diagrams.md

```diff
@@ -222,4 +222,12 @@ flowchart TD
 {% endmermaid %}{% endraw %}
 ```
 {% endnavtab %}
-{% endnavtabs %}
\ No newline at end of file
+{% endnavtabs %}
+
+## Export a Mermaid diagram from the docs
+
+You can export a Mermaid diagram from the Kong docs using the Mermaid live editor. 
+
+1. Copy the Mermaid diagram from the Kong doc file. You can find the doc source file by clicking **Edit this page** at the top right corner of the doc.
+1. Paste the Mermaid diagram into the [Mermaid live editor](https://mermaid.live/edit).
```

Comment:
```suggestion
1. Paste the Mermaid diagram into the [Mermaid live editor](https://mermaid.live/edit#pako:eNpl0D1vwjAQBuC_El3XDFQCRDK2VdWhXaDqUHm52Gew5I_o4lBSlP9eh4Dkiu107_PecGeQQRHUoG34kQfkWLxvoQRH7NCoFJyFLwoB8UCOBNRpbLBLU5ntv5ANNpa6CVwKU6SDj6_ojB3m3hvZI0Uj8Vq-mZ35vV5-XLWnLGzZOOThOdjAM3jQWt-Dp8CKOGdL3Kxpcy8_6RT_uWol18vMWeMpB1XVLORCwJSPwo_pNdjHsBu8hDpyTyX0rcJILwb3jA5qjbZLW1ImBv6Yn3v5cQkt-u8Qbmb8A-8_eHU).
```
It's ugly, but it gets them the right config/styling. If you just link to mermaid.live, they will get default mermaid styling.

Alternatively, you can add a step to paste in the following config in the config tab:

```
{
  "theme": "base",
  "themeVariables": {
    "fontFamily": "Helvetica",
    "fontSize": "15px",
    "primaryColor": "#fff",
    "primaryBorderColor": "#4a86e8",
    "primaryTextColor": "#495c64",
    "lineColor": "#99b0c0"
  }
}
```

Reply:
I like the ugly link, and maybe an explanation that it includes our styles.